programs/uxd/tests/integration_tests/api/program_uxd/accounts/create_identity_depository_keys.rs

```diff
@@ -0,0 +1,22 @@
+use solana_program::pubkey::Pubkey;
+
+pub struct IdentityDepositoryKeys {
+    pub depository: Pubkey,
+    pub collateral_vault: Pubkey,
+}
+
+pub fn create_identity_depository_keys() -> IdentityDepositoryKeys {
+    let depository =
+        Pubkey::find_program_address(&[uxd::IDENTITY_DEPOSITORY_NAMESPACE.as_ref()], &uxd::id()).0;
```

Comment:
All the PDA gen could be factorised somewhere for reuse

Reply:
(one by one)

Reply:
Maybe i'm miss understanding the comment, but currently each PDA is only generated once already, the function "create_**_keys" find all the PDAs and then everywhere in the code we just use the return value of that creation step

Reply:
the PDA generation code is never duplicated, for example the "controller" is being generated only in a single function, nowhere else.

If one day I need to re-find the controller in a different function i'll definitly create a helper function for this maybe, is that what you meant?

Reply:
Merging for now, I can apply this feedback in next PR, let me know if I miss understood